File: JAVA/isBinary.java
import java.util.Scanner;
public class isBinary {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int[] a = new int[100];
        int n;
        boolean flag;
        flag = true;
        System.err.println("Enter the length of the Binary Number:");
        n= s.nextInt();
        System.out.println("Enter the binary number by giving spave between each:");
        for(int i =  0; i<n;i++){
            a[i]=s.nextInt();
            if(a[i]>1 || a[i]<0){
                flag= false;
            }
        }
        if(flag){
            System.out.println("The entered value is binary.");
        }else{
            System.out.println("Its not a binary number.");
        }
        s.close();
    }
}

File: C++/isBinary.cpp
#include<iostream>
using namespace std;
#define MAX 1000
int main(){
    int a[MAX];
    int n, flag=0;
    
    cout<<"Enter the length of the binary number:";
    cin>>n;
    cout<<"Enter the binary number giving space between them:";
    for (int i=0; i<n;i++){
        cin>>a[i];
        if(a[i]>1 || a[i]<0){
            flag=1;
        }
    }
    if(flag==0){
        cout<<"Its a binary number"<<endl;
    }else{
        cout<<"Its not a binary number"<<endl;
    }
    return 0;
}